datafusion/physical-plan/src/aggregates/group_values/single_group_by/primitive.rs

```diff
@@ -85,7 +85,7 @@ pub struct GroupValuesPrimitive<T: ArrowPrimitiveType> {
     ///
     /// We don't store the hashes as hashing fixed width primitives
     /// is fast enough for this not to benefit performance
-    map: HashTable<usize>,
+    map: HashTable<(usize, u64)>,
```

Comment:
it seems like the comment right above this line is no longer accurate and needs to be updated as this PR exactly saves the hashes

Reply:
Addressed.